test/integration/config/index.tap.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2022 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const Config = require('../../../lib/config')
+const tap = require('tap')
+
+tap.test('Configuration Library', function (harness) {
+  harness.autoend()
+
+  let originalConsoleError
+
+  harness.beforeEach(function () {
+    // Make sure we don't pollute our logs
+    originalConsoleError = global.console.error
+    global.console.error = function () {}
+  })
+
+  harness.afterEach(function () {
+    // Restore so we don't have a knock-on effect with other test suites
+    global.console.error = originalConsoleError
+  })
+
+  harness.test('it should gracefully handle errors on instantiation', function (test) {
```

Comment:
Not sure we have a strong opinion on how to write the descriptions, but I usually omit the subject of the sentence.

Also, we still call the parameter `t` even in subtests, it's not a problem if it's shadowed.